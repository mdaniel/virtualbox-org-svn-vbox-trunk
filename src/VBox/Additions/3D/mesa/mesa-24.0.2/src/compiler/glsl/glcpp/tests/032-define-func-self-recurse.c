#define foo(a) foo(2*(a))
foo(3)
