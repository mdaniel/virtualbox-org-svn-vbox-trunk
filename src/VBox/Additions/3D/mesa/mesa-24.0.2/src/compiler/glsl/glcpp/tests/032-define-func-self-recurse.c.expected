
foo(2*(3))
